Mathlib/MeasureTheory/Function/SimpleFunc.lean

```diff
@@ -919,27 +919,28 @@ theorem restrict_const_lintegral (c : ℝ≥0∞) {s : Set α} (hs : MeasurableS
     ((const α c).restrict s).lintegral μ = c * μ s := by
   rw [restrict_lintegral_eq_lintegral_restrict _ hs, const_lintegral_restrict]
 
+@[gcongr]
+theorem lintegral_mono_fun {f g : α →ₛ ℝ≥0∞} (h : f ≤ g) : f.lintegral μ ≤ g.lintegral μ := by
+  refine Monotone.of_left_le_map_sup (f := (lintegral · μ)) (fun f g ↦ ?_) h
+  change ((pair f g).map Prod.fst).lintegral μ ≤ ((pair f g).map fun p ↦ p.1 ⊔ p.2).lintegral μ
```

Comment:
What's the `change` doing? Please try to replace it

Reply:
I turned this into `calc` and removed one defeq (ab)use. For the other one, we would need more `rfl` lemmas to rewrite and I think that this is an unrelated refactor (this defeq abuse was in the original proof).